chopper/lib/src/annotations.dart

```diff
@@ -387,7 +387,7 @@ class PartFile {
 @immutable
 @Deprecated('Use PartFile instead')
 class FileField extends PartFile {
-  const FileField([String name]) : super(name);
+  const FileField([String? name]) : super(name);
```

Comment:
since null safety version will introduce breaking change
we could remove this deprecated class

Reply:
We can pass on every deprecated API in an other PR

Reply:
Good idea!

Reply:
Cool, keep it as it.